server/src/pages/shot/share-buttons.js

```diff
@@ -86,20 +85,21 @@ class ShareButtonPanel extends React.Component {
       className += " share-panel-with-notification";
     }
     return <div id="share-buttons-panel" className={className}>
-      <div className="wrapper row-space">
+      <div className="wrapper row-wrap share-buttons">
         <a title="Share to Facebook wall or message" onClick={ this.onClickShareButton.bind(this, "facebook") } target="_blank" href={ "https://www.facebook.com/sharer/sharer.php?u=" + encodeURIComponent(this.props.shot.viewUrl) }>
-          <img src={ this.props.staticLink("/static/img/share-facebook.svg") } />
+          <img src={ this.props.staticLink("/static/img/btn-fb.svg") } />
```

Comment:
Nit: do we want `alt=""` attributes on these `<img>` tags?
Quick fix, although probably outside the scope of this PR since they weren't there originally either.